app/ethante/eip712.go

```diff
@@ -0,0 +1,321 @@
+// Copyright 2021 Evmos Foundation
```

Comment:
@facs95 can you migrate the files to `app/ante/evm` instead?

Reply:
Done